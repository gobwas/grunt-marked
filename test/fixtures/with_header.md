---
title: some title
/---


Test
====

```javascript
var a = 10;
```