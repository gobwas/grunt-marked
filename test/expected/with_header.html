<h1 id="test">Test</h1>
<pre><code class="lang-javascript"><span class="hljs-attribute">var a </span>=<span class="hljs-string"> 10;</span>
</code></pre>
